From dcc37de15a2650887cb7e39a94de890522cae839 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Fri, 26 Aug 2022 17:04:50 +0300
Subject: [PATCH] [SW-0] gaudi3: export: remove legacy function

'secureboot_verify_image()' function is a legacy function and obsolete
for gaudi3. Hence the same can be removed from arcboot export list.

Change-Id: Ief52405dc1ae6e80fc41602441357be1b636e8a5
---

diff --git a/arch/gaudi3/arc/arcboot_rom.txt b/arch/gaudi3/arc/arcboot_rom.txt
index 1508941..c0ba611 100644
--- a/arch/gaudi3/arc/arcboot_rom.txt
+++ b/arch/gaudi3/arc/arcboot_rom.txt
@@ -1,7 +1,6 @@
 # put list of symbols, one per line, to be exported
 dma_wait_finish
 security_i2c_s_enable
-secureboot_verify_image
 dma_copy_image
 spi_flash_readl
 dma_copy_image_v2
